src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +26,34 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todoModal?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
```

Comment:
The method naming convention is not followed here. The event handler onCloseButtonClick should be named handleButtonClick or handleButtonCloseClick according to the [naming conventions](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) in the checklist.